src/schema/v2/me/collection.ts

```diff
@@ -32,6 +32,9 @@ export const CollectionType = new GraphQLObjectType<any, ResolverContext>({
               .value,
           },
           page: { type: GraphQLInt },
+          forSale: { type: GraphQLBoolean },
+          priceMin: { type: GraphQLInt, description: "In USD Dollars" },
```

Comment:
in backend apps, we do have a pattern of naming fields w/ major/minor (to denote major or minor units, dollars vs. cents), as well as '_usd' suffix if it's been exchanged (as you can tell from the endpoint mapping below).

Not sure if we have that pattern on the FE really - should these be `priceMinMajorUSD` or something like that? (with money fields it's maybe better to be unambiguous)